packages/client/src/arpa_reporter/views/UploadsView.vue

```diff
@@ -215,7 +218,7 @@ export default {
         {
           label: 'EC Code',
           field: 'ec_code',
-          tdClass: (row) => (!row.ec_code ? 'table-danger' : undefined),
+          tdClass: (row) => (!row.ec_code ? 'usdr-table-danger' : undefined),
```

Comment:
Note to reviewer: before/after screenshots

The following will be addressed in separate PRs
- Accessible name
- Table heard text should not be empty

## Color Contrast - Before
![image](https://github.com/user-attachments/assets/b7cefc4f-4d2d-4cf1-8f62-78945f377165)

![image](https://github.com/user-attachments/assets/a0d30ece-7e77-4d16-afdc-7cb3adc79891)

![image](https://github.com/user-attachments/assets/0d1b5ffe-f7cd-4d16-8959-936d700934b2)

![image](https://github.com/user-attachments/assets/a421d216-90af-4c59-af9c-2e307cba0e32)


## Color Contrast - After
![image](https://github.com/user-attachments/assets/93dd9a67-d7ce-4205-82b7-8f188fddaed7)